docs/ccoctl.md

```diff
@@ -335,4 +335,89 @@ For alibaba cloud,  the CCO utility (`ccoctl`) binary will create credentials Se
    ```
    where:
    - `name` is the name used to tag any cloud resources that are created for tracking. 
-   - `region` is the Alibaba Cloud region in which cloud resources will be created.   
\ No newline at end of file
+   - `region` is the Alibaba Cloud region in which cloud resources will be created.   
+
+## Nutanix
+
+This is a guide for using manual mode on Nutanix, for more info about manual mode, please refer to [cco-mode-manual](https://github.com/openshift/cloud-credential-operator/blob/master/docs/mode-manual-creds.md).
+
+For Nutanix,  the CCO utility (`ccoctl`) binary will create credentials Secret manifests for the OpenShift installer.
+
+### Procedure
+
+1. Extract the list of CredentialsRequest custom resources (CRs) from the OpenShift Container Platform release image:
+
+   ```bash
+   $ oc adm release extract --credentials-requests --cloud=nutanix --to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
+   ```
+
+   >  step 2&3 are only needed when preparing for upgrading clusters with manually maintained credentials. When doing a fresh installation please skip step 2&3**
+
+2. For each CredentialsRequest CR in the release image, ensure that a namespace that matches the text in the spec.secretRef.namespace field exists in the cluster. This field is where the generated secrets that hold the credentials configuration are stored.
```

Comment:
In product docs, probably makes sense to include a command to list all namespaces so they can check against that

Reply:
Good point. Will add a `oc get namespace` statement to the sentence.